dev-tools/packaging/templates/windows/install-service.ps1.tmpl

```diff
@@ -6,12 +38,37 @@ if (Get-Service {{.BeatName}} -ErrorAction SilentlyContinue) {
   sc.exe delete {{.BeatName}}
 }
 
+# We need to support a new default path for the data folder, ideally
+# automatically detecting if the old one is used and keeping it
+
 $workdir = Split-Path $MyInvocation.MyCommand.Path
```

Comment:
Nit.  Any chance we could change from `workdir` to `WorkDir` so it matches all the other variables (HomePath, LogsPath, etc)?

Reply:
Done: https://github.com/elastic/beats/pull/43995/commits/5cd87f5142ddc6820b4596fe5f58cfbc527a11db